### DISCUSSION TITLE
RPC fibonacci python tutorial

### DISCUSSION THREAD

**Discussion by MisterCodeRalf** [rabbitmq/rabbitmq-website#1626] [General]
Leaving connections open is not a good idea. This may lead to overflows:
 
in the rpc_client.py file in the def __init__(self) a close should be defined: 

def close(self):
        self.connection.close()

this should then be called at the end of the  rpc_client.py file:

fibonacci_rpc.close()

**Comment by michaelklishin** [comment]
These are tutorials. They cannot demonstrate everything at once. I would argue that using short lived connections in tutorials is/would be a bigger issue for us because people copy code into production and create connection churn, which is so problematic it has two dedicated sections in various doc guides.
